src/components/LHNOptionsList/OptionRowLHNData.tsx

```diff
@@ -63,43 +16,45 @@ const defaultProps = {
  * re-render if the data really changed.
  */
 function OptionRowLHNData({
-    isFocused,
+    isFocused = false,
     fullReport,
     reportActions,
-    personalDetails,
-    preferredLocale,
+    personalDetails = {},
+    preferredLocale = CONST.LOCALES.DEFAULT,
     comment,
-    policy,
+    policy = null,
```

Comment:
No need for null default values:
```suggestion
    policy,
```